ci/docs/continuous_integation.md

```diff
@@ -10,13 +10,15 @@ HyperShift official [Documentation Guide](https://hypershift-docs.netlify.app/).
 
 ### Pre-requisites:
 
-- Install [aws](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) cli
+- Install [AWS](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) cli
```

Comment:
`aws`, as this is the command line binary, not the service.

Reply:
fixed